docs/usage.md

```diff
@@ -1,284 +1,375 @@
-| `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+Running the pipeline involves three steps:
 
-An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
+1. Prepare a samplesheet
+2. Gather all required references
+3. Supply samplesheet and references, and run the command
 
-## Running the pipeline
+#### Samplesheet
 
-The typical command for running the pipeline is as follows:
+A samplesheet is used to pass the information about the sample(s), such as the path to the FASTQ files and other meta data (sex, phenotype, etc.,) to the pipeline in csv format.
 
-```console
-nextflow run nf-core/raredisease --input samplesheet.csv --genome GRCh37 -profile docker
-```
+nf-core/raredisease will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The pedigree information in the samplesheet (sex and phenotype) should be provided as they would be for a [ped file](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format) (i.e. 1 for male, 2 for female, other for unknown).
 
-This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles.
+| Fields        | Description                                                                                                                                                                            |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample`      | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `lane`        | Used to generate separate channels during the alignment step.                                                                                                                          |
+| `fastq_1`     | Absolute path to FASTQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                         |
+| `fastq_2`     | Absolute path to FASTQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                         |
+| `sex`         | Sex (1=male; 2=female; other=unknown).                                                                                                                                                 |
```

Comment:
Any reason to go for `1`/`2` compared to `XY`/`XX`?

Reply:
This notation is used by plink https://zzz.bwh.harvard.edu/plink/data.shtml#ped